deployment/windows/UAC.nsh

```diff
@@ -0,0 +1,299 @@
+/*** UAC Plug-in ***
```

Comment:
Usually just setting [`RequestExecutionLevel admin`](http://nsis.sourceforge.net/Reference/RequestExecutionLevel) in main NSIS script works good. Not sure if separate plugin required there.

Reply:
I think the advantage of the UAC plugin is that you can decide request UAC only if the user decides to install for all users, or continue without UAC otherwise. If you request UAC, the plugin will launch a second instance of the installer with elevated privileges.

Reply:
I'm not 100% sure, but UAC is required to install [anything](https://superuser.com/questions/1031275/windows-allow-normal-non-elevated-programs-to-write-to-program-files-director) into default "C:/Program FIles/"

Reply:
Right, that's why the UAC plugin will launch a second installer with elevated privileges if you choose to install for your users. If you choose to install for only yourself, it will install under your home account and not need UAC. This is not implemented yet, but it is easy to add that page to the installer.

Reply:
@PombeirP got it, thanks for details!